backend/tests/unit/api/Helpers/VisualStudioProvider.cs

```diff
@@ -0,0 +1,16 @@
+using System.IO;
+using System.Linq;
+
+public static class VisualStudioProvider
```

Comment:
this looks too specific. What is this used for?

Reply:
Yeah, will comment/rename. This finds the sln file directory (root) of the project.